widget/progressbarinfinite.go

```diff
@@ -113,7 +113,7 @@ func (p *infProgressRenderer) start() {
 	}
 }
 
-// Stop the infinite progress goroutine and sets value to the Max
+// Stop the infinite progress bar background thread from to updating it
```

Comment:
```suggestion
// Stop the infinite progress bar background thread from updating it
```

Or better yet:
```suggestion
// Stop the background thread from updating the infinite progress bar
```

Reply:
Note: this in an unexported function, so the doc is not developer-facing

Reply:
Oh, good suggestion. Indeed, that’s why I made sure to keep the internal details here 🙂